dbusmock/templates/networkmanager.py

```diff
@@ -26,8 +26,9 @@
 
 
 BUS_NAME = 'org.freedesktop.NetworkManager'
-MAIN_OBJ = '/org/freedesktop/NetworkManager'
-MAIN_IFACE = 'org.freedesktop.NetworkManager'
+MAIN_OBJ = '/org/freedesktop'
```

Comment:
Oh, so `/org/freedesktop` is an actually valid path in NM, I just saw in d-feet. Sorry about that then. This certainly looks very strange, I had expected the main object /o/fd/NM to be the object manager too.

MAIN_OBJ is the first/main object that dbusmock creates and the only object that will remain after you call `Reset()`. While `Reset()` is API, using the latter on a template certainly doesn't make much sense, so I think we can live with that change wrt. backwards compatibility.

So I guess that part is okay then.